packages/suite-build/utils/constants.ts

```diff
@@ -1,7 +1,8 @@
-export const PROJECTS = ['web', 'desktop'] as const;
+export const PROJECTS = ['web', 'desktop', 'landing'] as const;
 export type Project = typeof PROJECTS[number];
 
 export const DEV_PORTS: { [key in Project]: number } = {
     web: 8000,
     desktop: 8000,
+    landing: 3000,
```

Comment:
Was there some discussion why it should run on a different port?

When I click on the `Trezor suite for web` button, it does not redirect to `localhost:8000`, so it does not make sense for me.